content/en/docs/contributing/release-process.md

```diff
@@ -27,181 +27,211 @@ following conditions:
 3. You need to have the GitHub `write` permission on the cert-manager project.
    To check that you have the `write` role, [get a personal access
    token](https://github.com/settings/tokens) and run:
-     ```sh
-     GH_USER=maelvls
-     curl -sH "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/jetstack/cert-manager/collaborators/$GH_USER/permission
-     ```
-     If your permission is `write` or `admin`, then you are good to go:
-     ```json
-     {
-      "permission": "write",
-      "user": {...}
-     }
-     ```
 
-    To request the `write` permission on the cert-manager project, [open a
-   PR](https://github.com/jetstack/platform-board/pulls/new) with a link to
-   here.
+    ```sh
+    go install github.com/cli/cli/cmd/gh@latest
```

Comment:
This tag isn't go installable, either find one that is or use curl to install a release

Reply:
Thanks! Fixed in c746b4d786e4d1161b9dea011f751cf563dfd3dc